src/tests/unit/__snapshots__/contact-info.test.tsx.snap

```diff
@@ -0,0 +1,36 @@
+// Vitest Snapshot v1
+
+exports[`ContactInfoInputs > Should NOT wrap the input fields if it is NOT inline 1`] = `
+<div
+  data-cy="contact-info-input"
+>
+  <p>
+    One input component
+  </p>
+  <p>
+    Another input component
+  </p>
+</div>
+`;
```

Comment:
I do not understand the changes here. It seems as if they duplicate what is already in [`ContactInfoInputs.tsx.snap`](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/579/files#diff-bc6549b47347bee1fec80f79777e22f5e97b470a3a3c1a568d9be6d2a380061f). Is the former obsolete?

Reply:
No there is two assertions:
One that asserts that we can show the children wrapped in inline wrapper and one that we can show the children *without* the inline wrapper